editor/connections_dialog.cpp

```diff
@@ -282,10 +282,10 @@ List<MethodInfo> ConnectDialog::_filter_method_list(const List<MethodInfo> &p_me
 	bool check_signal = compatible_methods_only->is_pressed();
 	List<MethodInfo> ret;
 
-	List<Pair<Variant::Type, StringName>> effective_args;
+	LocalVector<Pair<Variant::Type, StringName>> effective_args;
```

Comment:
```suggestion
	LocalVector<Pair<Variant::Type, StringName>> effective_args;
	effective_args.reserve(p_signal.arguments.size() - unbind);
```
See https://github.com/godotengine/godot/pull/99936#issuecomment-2515610193